pysumma/Simulation.py

```diff
@@ -21,7 +22,7 @@ def __init__(self, filepath):
         self.meta_type = FileManagerOption(self, 'meta_type')
         self.meta_force = FileManagerOption(self, 'meta_force')
         self.meta_localpar = FileManagerOption(self, 'meta_localpar')
-        self.OUTPUT_CONTROL = FileManagerOption(self, 'OUTPUT_CONTROL')
```

Comment:
@arbennett I think we have to edit SUMMA Test Case before merging this commit. Do you have updated Test Cases and SUMMA version?